integration/js/steps/JoinVideoTestMeetingStep.js

```diff
@@ -0,0 +1,47 @@
+const {KiteTestError, Status} = require('kite-common');
+const AppTestStep = require('../utils/AppTestStep');
+
+class JoinVideoTestMeetingStep extends AppTestStep {
```

Comment:
Dont we already have a Step that joins a meeting?

Reply:
Yes, but I need a customized method to pass in the attendeeId and meetingTitle so that we can enter them in the UI input boxes.